test/admin/customUserData.test.js

```diff
@@ -0,0 +1,67 @@
+const StitchMongoFixture = require('../fixtures/stitch_mongo_fixture');
+
+import { BSON } from 'mongodb-extjson';
+import { buildAdminTestHarness, extractTestFixtureDataPoints } from '../testutil';
+
+function createSampleMongodbService(services) {
+  return services.create({
+    type: 'mongodb',
+    name: 'mdb',
+    config: {
+      uri: 'mongodb://localhost:26000'
+    }
+  });
+}
+
+describe('Custom User Data', () => {
+  let test = new StitchMongoFixture();
+  let th;
+  let customUserData;
+  let services;
+
+  beforeAll(() => test.setup());
+  afterAll(() => test.teardown());
+
+  beforeEach(async() => {
+    const { apiKey, groupId, serverUrl } = extractTestFixtureDataPoints(test);
+    th = await buildAdminTestHarness(true, apiKey, groupId, serverUrl);
+    customUserData = th.app().customUserData();
+    services = th.app().services();
+  });
+
+  afterEach(async() => th.cleanup());
+
+  it('should be able to get custom user data', async() => {
+    let data = await customUserData.get();
+    expect(data.mongo_service_id).toBeDefined();
+    expect(data.database_name).toBeDefined();
+    expect(data.collection_name).toBeDefined();
+    expect(data.user_id_field).toBeDefined();
+    expect(data.enabled).toBeDefined();
+  });
+
+  it('should be able to update custom user data', async() => {
+    const mongodbService = await createSampleMongodbService(services);
+
+    let newData = {
+      mongo_service_id: mongodbService._id,
+      enabled: true,
+      database_name: 'booboo',
+      collection_name: 'theFool',
+      user_id_field: 'vegeta'
+    };
+    let originalData = await customUserData.get();
+    expect(originalData.enabled).toBe(false);
+    expect(originalData.database_name).toBe('');
+    expect(originalData.collection_name).toBe('');
+
+    await customUserData.update(newData);
+
+    let updatedData = await customUserData.get();
```

Comment:
[nit] same with this, have a before that does the update etc. then have individual `it` for checking each field